user_manual/files/access_webdav.rst

```diff
@@ -58,6 +58,1028 @@ You can access files in Linux operating systems using the following methods.
+
+Problem
+^^^^^^^
+
+You cannot download more than 50 MB or upload large Files when the upload takes
+longer than 30 minutes using Web Client in Windows 7.
+
+Solution
+^^^^^^^^
+
+Workarounds are documented in the KB2668751_ article.
+
+
+Accessing files using cURL
+--------------------------
+
+Since WebDAV is an extension of HTTP cURL can be used to script file operations.
+
+To create a folder with the current date as name:
+
+.. code-block:: bash
+
+	$ curl -u user:pass -X MKCOL "https://example.com/nextcloud/remote.php/dav/files/USERNAME/$(date '+%d-%b-%Y')"
+
+To upload a file ``error.log`` into that directory:
+
+.. code-block:: bash
+
+	$ curl -u user:pass -T error.log "https://example.com/nextcloud/remote.php/dav/files/USERNAME/$(date '+%d-%b-%Y')/error.log"
+
+To move a file:
+
+.. code-block:: bash
+
+	$ curl -u user:pass -X MOVE --header 'Destination: https://example.com/nextcloud/remote.php/dav/files/USERNAME/target.jpg' https://example.com/nextcloud/remote.php/dav/files/USERNAME/source.jpg
+
+To get the properties of files in the root folder:
+
+.. code-block:: bash
+
+	$ curl -X PROPFIND -H "Depth: 1" -u user:pass https://example.com/nextcloud/remote.php/dav/files/USERNAME/ | xml_pp
+	<?xml version="1.0" encoding="utf-8"?>
+    <d:multistatus xmlns:d="DAV:" xmlns:oc="http://nextcloud.org/ns" xmlns:s="http://sabredav.org/ns">
+      <d:response>
+        <d:href>/nextcloud/remote.php/dav/files/USERNAME/</d:href>
+        <d:propstat>
+          <d:prop>
+            <d:getlastmodified>Tue, 13 Oct 2015 17:07:45 GMT</d:getlastmodified>
+            <d:resourcetype>
+              <d:collection/>
+            </d:resourcetype>
+            <d:quota-used-bytes>163</d:quota-used-bytes>
+            <d:quota-available-bytes>11802275840</d:quota-available-bytes>
+            <d:getetag>"561d3a6139d05"</d:getetag>
+          </d:prop>
+          <d:status>HTTP/1.1 200 OK</d:status>
+        </d:propstat>
+      </d:response>
+      <d:response>
+        <d:href>/nextcloud/remote.php/dav/files/USERNAME/welcome.txt</d:href>
+        <d:propstat>
+          <d:prop>
+            <d:getlastmodified>Tue, 13 Oct 2015 17:07:35 GMT</d:getlastmodified>
+            <d:getcontentlength>163</d:getcontentlength>
+            <d:resourcetype/>
+            <d:getetag>"47465fae667b2d0fee154f5e17d1f0f1"</d:getetag>
+            <d:getcontenttype>text/plain</d:getcontenttype>
+          </d:prop>
+          <d:status>HTTP/1.1 200 OK</d:status>
+        </d:propstat>
+      </d:response>
+    </d:multistatus>
+
+
+.. _KB2668751: https://support.microsoft.com/kb/2668751
+.. _KB2123563: https://support.microsoft.com/kb/2123563
+.. _WebDAV Navigator: http://seanashton.net/webdav/
+.. _Android devices: https://play.google.com/store/apps/details?id=com.schimera.webdavnavlite
+.. _iPhones: https://itunes.apple.com/app/webdav-navigator/id382551345
+.. _BlackBerry devices: http://appworld.blackberry.com/webstore/content/46816
```

Comment:
```suggestion
.. _BlackBerry devices: http://appworld.blackberry.com/webstore/content/46816

```

To fix `Explicit markup ends without a blank line; unexpected unindent.` warning.